tests/samples/matmul_fill_static.mlir

```diff
@@ -1,9 +0,0 @@
-func.func @matmul_static(%lhs : tensor<128x256xf32>,
```

Comment:
Keep these inputs that are needed to run the transform dialect scripts. https://github.com/openxla/iree/blob/main/compiler/src/iree/compiler/Codegen/LLVMGPU/test/transform_dialect_eliminate_gpu_barriers.mlir is an example of how you can use `iree-opt` + transform dialect script to keep the examples and transform dialect script together.